src/css/_acquia_lift.navbar.styles.scss

```diff
@@ -0,0 +1,397 @@
+/**
+ * @file
+ * Styles for integrating Personalize with the Navbar module.
+ */
+
+/**
+ * Personalize top-level icon.
+ */
+.navbar-bar .navbar-icon-acquia-lift:before {
+  background-image: url("../images/bebebe/rocket.svg");
+}
+
+.no-svg .navbar-bar .navbar-icon-acquia-lift:before {
+  background-image: url("../images/bebebe/rocket.png");
+}
+
+.navbar-bar .navbar-icon-acquia-lift {
+  &:active:before, &.navbar-active:before {
+    background-image: url("../images/ffffff/rocket.svg");
+  }
+}
+
+.no-svg .navbar-bar .navbar-icon-acquia-lift {
+  &:active:before, &.navbar-active:before {
+    background-image: url("../images/ffffff/rocket.png");
+  }
+}
+
+/**
+ * Personalize second-level icon.
+ */
+.navbar-box .navbar-icon-acquia-lift:before {
+  background-image: url("../images/707070/rocket.svg");
+}
+
+.no-svg .navbar-box .navbar-icon-acquia-lift:before {
+  background-image: url("../images/707070/rocket.png");
+}
+
+#navbar-administration {
+  .acquia-lift-active {
+    font-weight: bold;
+    .acquia-lift-campaign {
+      font-weight: bold;
+    }
+  }
+  .acquia-lift-campaigns .navbar-box {
+    display: inline-block;
+    width: 100%;
+  }
+  .navbar-tray-vertical .acquia-lift-campaigns .navbar-box .acquia-lift-campaign {
+    margin-right: 0;
+  }
+  &.navbar-oriented .acquia-lift-campaign {
+    padding-right: 0; /* LTR */
+  }
+}
+
+/* Menu wrapper container */
+#navbar-administration {
+  .navbar-tray-vertical .navbar-twisty {
+    > .menu-wrapper {
+      display: none;
+    }
+    &.open > .menu-wrapper {
+      display: block;
+    }
+  }
+  .acquia-lift-menu-item {
+    margin: {
+      top: .88em;
+      bottom: .88em;
+    }
+  }
+  .navbar-level-1 {
+    .menu-wrapper > .#{$menu-class},
+    .#{$menu-class} {
```

Comment:
This is awesome. And $menu-class is something that can go in _variables.scss to make it available globally.